azure/notebooks/Azure-MNMG-RF.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    dask.config.set({"logging.kubernetes": "info",
</pre><p>Why are we logging k8s info?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
We do not need this. My bad. This is an artifact I missed. Should remove this from #127  as well.